File: app/problems/max_number/runnable-code/execute.cpp
#include <iostream>
#include <cmath>
#include <string>
#include <algorithm>
#include <vector>

using namespace std;

##USER_CODE_HERE##

int main() {
    int size, num;
    cin >> size;
    cin >> num;
    vector<int> arr(size);
    for (int i = 0; i < size; i++) {
        cin >> arr[i];
    }
    cout << max_number(arr) << endl;
    return 0;
}

File: app/problems/max_number/runnable-code/execute.js
let intput = "";

##USER_CODE_HERE##

process.stdin.on("data", (chunk) => {
  if (chunk !== null) {
    intput += chunk;
  }
});

process.stdin.on("end", () => {
  const inputArr = intput.split("\n");
  const arr = inputArr.slice(1);
  console.log(max_number(arr));
});